less/index.less

```diff
@@ -48,12 +48,204 @@ table {
 	border-spacing: 0;
 }
 
+
+//variables
+@border-color: #90cfc5; //seafoam green
+@text-header:  'Lobster', cursive;
+@text-int:  'Lora', serif;
+@text-main:  'Raleway', sans-serif;
+
+//mixins
+.center() {
+	display: flex;
+	justify-content: center;
+	align-items: center;
+}
+.space-between(){
+	display: flex;
+	flex-direction: column;
+	justify-content: space-between;
+}
+.column(){
+	display: flex;
+	flex-direction: column;
+	align-items: stretch;
+	
+}
+.top(){
+	display: flex;
+	flex-direction: column;
+	justify-content: space-between;
+	height: 15.5%;
+	padding: 8px 0;
+	border-top: 3px solid black;
+}
+.middle(){
+	display: flex;
+	flex-direction: column;
+	justify-content: space-between;
+	height: 50.5%;
+	padding: 8px 0;
+	border-top: 3px solid black;
+}
+.bottom(){
+	display: flex;
+	flex-direction: column;
+	justify-content: space-between;
+	height: 25%;
+	padding: 8px 0;
+	border-top: 3px solid black;
+}
+
 /* Set every element's box-sizing to border-box */
 * {
     box-sizing: border-box;
 }
 
 html, body {
-    height: 100%;
+	height: 100%;
+	font-size: 62.5%;
+}
+body {
+	max-width: 500px;
+	width: 100%;
+	margin: 0 auto;
+	border: 16px solid @border-color;
+	min-height: 500px; 
+	@media(max-width: 500px){
+		max-width: 250px;
+	}
 }
+h3 {
+	font-family: @text-main;
+	font-weight: 600;
+	font-size: .8rem;
+}
+.bold{
+	font-family: @text-main;
+	font-weight: 600;
+	font-size: .7rem;
+}
+.text{
+	font-family: @text-main;
+	font-weight: 400;
+	font-size: .7rem;
+}
+.location{
+	font-family: @text-main;
+	font-weight: 300i;
+	font-size: .5rem;
+}
+
+.container{
+	display:flex;
+	justify-content: space-evenly;
+	height: 100%;
+	.left-content{
+		width: 33%;
+		.column();
+		padding: 40px 0 30px 0;
+		.left-header{
+			height: 25%;
+			display: flex;
+			justify-content: flex-end;
+			 h1{
+				font-family: @text-header; 
+				font-size: 3.2rem;
+				@media(max-width: 500px){
+					font-size: 2.27rem;
+				}
+
+			}
+		}
+		.contact{
+			.top();
+				p{
+				font-family: @text-main;
+				font-weight: 600;
+				font-size: .7rem;
+			}
+		}
+		.education{
+			.middle();
+		}
+		.skills{
+			.bottom();
+		}
+	}
+	.middle-content{
+		width: 16.5%;
+		.column();
+		.middle-header{
+			margin-top: 25px;
+			padding: 15px 0;
+			.center();
+			border-top: 5px solid black;
+			border-bottom: 5px solid black;
+			border-radius: 5px;
+			@media(max-width: 500px){
+				margin-top: 15px;
+			}
+			h2{
+				font-family: @text-int; 
+				font-size: 2.2rem;
+				@media(max-width: 500px){
+					font-size: 3rem
+				}
+				
+			}
+		}
+		.middle-img{
+			padding: 38px 0;
+			.center();
+			flex-flow: column;
+			.divider-box1{
+				height: 48px;
+				border-right: 3px dotted gray;
+			}
+			.divider-box2{
+				height: 168px;
+				border-right: 3px dotted gray;
+			}
+			@media(max-width: 500px){
+				display: none;
+			}
+		}
+
+	}
+	.right-content{
+		width: 33%;
+		.column();
+		padding: 40px 0 30px 0;
```

Comment:
You could shorten these paddings from _TRBL_ to _T LR B_ with `padding: 40px 0 30px;`